react/features/base/modal/components/JitsiKeyboardAvoidingView.js

```diff
@@ -45,19 +50,22 @@ const JitsiKeyboardAvoidingView = (
             contentContainerStyle,
             hasTabNavigator,
             hasBottomTextInput,
+            isModalPresentation,
             style
         }: Props) => {
-    const headerHeight = useHeaderHeight();
```

Comment:
@saghul for some reason this returns 0, so this hook was useless. I managed to find another way to get it. 
Right now everything looks good on devices without a top notch. I need to find a way to tackle devices that have a top notch, because the input feels too high when keyboard is visible. Please give it a test if have some time available.

Reply:
I tested and it looks and feels ok on my iPhone 12. Why do we care about the top when the keyboard shows from the bottom though?